obsidian.css

```diff
@@ -222,7 +222,7 @@ pre code, .cm-s-obsidian pre.HyperMD-codeblock {
 .cm-s-obsidian span.cm-formatting-list-ul {
   font-weight: bold;
 }
-ul { list-style: none; }
+ul { list-style: none !important; }
```

Comment:
It could be replaced with a more verbose selector if you prefer not to use `!important`.
```css
ul, ul ul, ol ul, ol ol ul, ol ul ul, ul ol ul, ul ul ul { list-style-type: none; }
```